lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -113,6 +120,51 @@
   </div>
 %endif
 
+%if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS') and section_data['access']['instructor']:
```

Comment:
The section above this has the same check: https://github.com/Stanford-Online/edx-platform/pull/296/files#diff-053b121559d9f6b574fca76d2f79b4adL88
do we need to keep them separate?
Also to be consistent with the rest of the file, use 2 spaces for indents